### PR TITLE
Monthly downloads badge added

### PR DESCRIPTION
**Pull request by mylapallilavanyaa**
Fixes # .

### Description

A few sentences describing the changes proposed in this pull request.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Comment by mingxin-zheng** [comment]
I just triggered the `update branch` to solve ci issues. Thanks!

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 **M**edical **O**pen **N**etwork for **AI**
 
+[![monai Downloads Last Month](https://assets.piptrends.com/get-last-month-downloads-badge/monai.svg 'monai Downloads Last Month by pip Trends')](https://piptrends.com/package/monai)
```
Comment (KumoLiu) [inline README.md:7]: Thanks for the PR, could you please move it after line17 to make the length of the two rows more symmetrical.
Screen Shot 2024-07-02 at 11 51 26

Comment (ericspod) [reply README.md:7]: Agreed. With the badge moved this would be fine to merge.

**Comment by KumoLiu** [comment]
Hi @mylapallilavanyaa, do you plan to help address the comments? Thanks!

**Review by KumoLiu — APPROVED** [review]

**Pull request merged**